### PR TITLE
ADD:  Notification actions

### PR DESCRIPTION
**Pull request by marcosrdz**
Super useful for the Apple Watch. It uses the explorer you have in ur settings

Simulator Screenshot - iPhone 16 Pro Max - 2024-10-26 at 21 05 15

## Release Notes

- **New Features**
	- Added localization support with `Localizable.xcstrings` for multiple languages.
	- Enhanced notification handling with new actions for viewing addresses and transactions in the browser.

- **User Interface Improvements**
	- Updated localized strings for the watchOS application, improving clarity and user interaction.

- **Project Configuration**
	- Adjusted project settings to support localization and enhance notification capabilities.

### CODE REVIEW

**Comment by marcosrdz** [comment]
https://github.com/BlueWallet/GroundControl/pull/289

File: ios/Localizable.xcstrings [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+          }
+        },
+        "fr" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Voir dans le navigateur"
+          }
+        }
+      }
+    },
```
[the remark this answers is not in the document]
Comment (sara554-beep) [reply ios/Localizable.xcstrings:42]: Puede ser que todo me salga en español

**Comment by sara554-beep** [comment]
Por favor autorizo hacer cambios

atte. sara silva

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**